public/models.ts

```diff
@@ -32,7 +32,7 @@ export interface ProjectData {
   readonly outputLocation: string;
 }
 
-export interface Timings extends Array<LineTiming> {}
+export type Timings = Array<LineTiming>;
```

Comment:
@irahopkinson how do I know when to use a type as opposed to an interface, like in this situation?

Reply:
You use a type here because the there was no additions to the base type `Array`, i.e. the empty definition `{}`. So a type definition here is enough to declare them as equivalent. Only use the interface to add or change something. ESLint can tell you this.